tex/gloss-hebrew.ldf

```diff
@@ -31,16 +29,10 @@ and may look very wrong.}
     \@calendar@hebrewfalse%
 	\fi}
 
-\define@boolkey{hebrew}[@xpg@hebrew@]{marcheshvan}[false]{%
-  \def\@tmpa{#1}%
-  \def\@tmptrue{true}%
-  \ifx\@tmpa\@tmptrue
-    \@xpg@hebrew@marcheshvantrue
-  \else
-    \@xpg@hebrew@marcheshvanfalse
-  \fi}
+\newif\if@xpg@hebrew@marcheshvan
+\@xpg@hebrew@marcheshvanfalse
 
-\setkeys{hebrew}{marcheshvan}
+\define@boolkey{hebrew}[@xpg@hebrew@]{marcheshvan}[true]{}
```

Comment:
`\define@key{hebrew}{marcheshvan}{\@xpg@hebrew@marcheshvantrue}` in addition?

Reply:
I think that's not necessary.

`\define@boolkey{hebrew}[@xpg@hebrew@]{marcheshvan}[true]{}`

means that you can also pass `marcheshvan` without explicit value (since `true` was defined as default in the second optional argument)

Reply:
You’re right, I hadn’t even thought of that.  But I just tried it now, and it works.  Should we document `[marcheshvan]` instead of `[marcheshvan=true]`, then?  The former feels more natural, and nobody is ever going to use `[marcheshvan=false]` in real life, so there’s no point in mentioning the option, in my opinion.

Reply:
I propose to as (default value = true) to indicate that, as for the other options.

Reply:
Sounds good, thanks for the suggestion.